SU2_CFD/src/numerics_direct_mean_hybrid.cpp

```diff
@@ -345,7 +354,7 @@ void CAvgGrad_Hybrid::AddTauSGETJacobian(su2double *val_Mean_PrimVar,
 	  tauSGET_mom[i][j] += -xi*mu[m][k]*nvec[m]*nvec[k]*delta[i][j];
 	}
 	tauSGET_mom[i][j] += -xi*mu[i][k]*nvec[j]*nvec[k];
-	tauSGET_mom[i][j] -= -xi*TWO3*mu[j][k]*nvec[i]*nvec[k];
+	//tauSGET_mom[i][j] -= -xi*TWO3*mu[j][k]*nvec[i]*nvec[k];
```

Comment:
I don't think this change is correct, if we're still using the deviatoric velocity.  Based on my derivations, I would expect the correct line to be:
```
tauSGET_mom[i][j] += -xi*mu[i][k]*nvec[j]*nvec[k]/3.0;
```

Reply:
Yes, I agree the Jacobian of tauSGET is incorrect.  My derivation used the full velocity gradient, not the deviatoric part, and subtracted out the trace of tauSGET.  That is where the commented out term came from.

In any case, I get two extra contributions: the line you had and

`tauSGET_mom[i][j] -= -xi*mu[k][i]*nvec[j]*nvec[k]/3.0;`

Both lines were added in 1ea2891, and I have a channel case running now to test.